src/resources/dev-envs/clion.md

```diff
@@ -1,80 +1,35 @@
-Use your university email address. After you complete the application form you will be prompted to create a JetBrains account. Your license will be tied to this account. You will recieve a confirmation email about your license application to your university email address.
-
-## Downloading CLion
-
-You can download CLion at: <https://www.jetbrains.com/clion/download>
+# Setting Up Visual Studio Community
 
+CLion is a commercial IDE from JetBrains. You can download CLion [here](https://www.jetbrains.com/clion/download).
 
 During installation, tick the "Add bin folder to PATH" and "Add Open Folder as Project" options.
+After completing the installation, you should be prompted to log into your JetBrains account.
 
-After completing the installation, you should be prompted to log into your JetBrains account.<br>
 If this doesn't happen, you can log in from the menu later after creating a project: Help -> Register...
 
-## Installing devtools
+## Installing a compiler
 
-CLion ships with MinGW GCC on Windows. If you want to install MSVC, use the Visual Studio Installer to install it. You will be able to use MSVC from CLion.
+CLion ships with MinGW GCC on Windows.
 
-## Creating a new project
+::: info
+If you want to install MSVC, use the Visual Studio Installer to install it. You will be able to use MSVC from CLion.
+:::
+
+## Getting Started
 
 After you installed and activated CLion with your JetBrains account, you should see this page:
 
-![alt text](../../public/assets/clion/clion-1.png)
+![CLion welcome menu](/assets/clion/clion-1.png)
 
-Click "New Project". Select C++ Executable, name your project and select a C++ standard. C++23 is a good choice if you don't know what to use.
+Click "New Project". Select C++ Executable, name your project and select a C++ standard (or select "C Executable" if you want to use C). C++23 is a good choice (or C23 for C) if you don't know what to use.
 
 The next thing you will see is this:
 
-![alt text](../../public/assets/clion/clion-2.png)
-
-If you have other toolchains like MSVC installed, they will appear here. If you don't just click "Ok".<br>
-Try and see if your setup works by pressing the green play button or press Shift + F10.<br>
-A terminal should pop up at the bottom that says "Hello, World!".
-
-## Useful settings in CLion
-
-* If you want to disable the one line JetBrains AI completions, create a file called `.noai` in the project's root directory.
-* Disable smart identation: Settings > Editor > Code Style > Uncheck: "Detect and use existing file idents for editing"
-* Configure your keybindings as you wish in the settings
-* Configure your code style like identation, line breaks, naming, etc. : Settings > Editor > Code Style > C/C++
+![CLion project wizard](/assets/clion/clion-2.png)
 
-## Using the debugger in CLion
+If you have other toolchains like MSVC installed, they will appear here. If you don't just click "Ok".
```

Comment:
The "other toolchains" part should probably be a note instead of part of the main text. Also, better working would be:

If you have other toolchains, such as MSVC, installed, they will appear here.